src/react-queries/hooks/useIsTradingActive.ts

```diff
@@ -44,12 +44,7 @@ export const useIsTradingActive = ({
     data: mimir,
     isLoading: isMimirLoading,
     refetch: refetchMimir,
-  } = useQuery({
-    ...reactQueries.thornode.mimir(),
-    queryKey: reactQueries.thornode.mimir().queryKey,
-    queryFn: enabled && assetId ? reactQueries.thornode.mimir().queryFn : skipToken,
```

Comment:
Removed `assetId` as `enabled` criteria, since this is now a shared query, I don't really see an issue in fetching either way (consumers can pass `enabled={false}` as they wish if they want to disable this) and `assetId` should not really be undefined either way, more like a types being too strict kind of thing

Reply:
Also see https://github.com/shapeshift/web/pull/9538 - `assetId` will never be undefined at types level (and well, runtime, even if nothing really changed there) after that PR, with the exception of `<AddLiquidityInput />` where it may during one or two renders